src/napari_stress/_surface.py

```diff
@@ -2,12 +2,69 @@
 
 import numpy as np
 import napari_process_points_and_surfaces as nppas
-from napari.types import LabelsData, SurfaceData, PointsData
+from napari.types import LabelsData, SurfaceData, PointsData, VectorsData
 from napari_stress._utils.frame_by_frame import frame_by_frame
 from napari_tools_menu import register_function
 
 import vedo
-import typing
+
+@register_function(menu="Points > Fit ellipsoid major axis to points (vedo, n-STRESS)")
+@frame_by_frame
+def fit_ellipsoid_to_pointcloud_points(points: PointsData, pvalue: float = 0.673) -> PointsData:
```

Comment:
I'm afraid the name "p-value" is confusing. How about nami g it something like "inside_points_fraction"?

Reply:
Fixed in https://github.com/BiAPoL/napari-stress/pull/90/commits/6741b2dc6ee0a9c217fc7d920360578cb0abc728